packages/base/core/src/svg/icons/concept/nutanix_concept.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
```

Comment:
I think `viewBox` values has changed. There's now an empty space under the paths. Not sure that it's a wanter behavior. Can cause alignment issues I think. Could you please adjust it?

Reply:
Following my previous comment, here is a screenshot of the empty area (since `viewbox` has changed).

![screenshot-icon](https://user-images.githubusercontent.com/428384/129669502-68b1a06c-79e1-4b4a-b0ff-55a8bd6816f3.png)

Could you please adjust it?

Reply:
I have reverted to original viewBox dimentions